src/ResourceManager/Automation/Commands.Automation/Az.Automation.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.0.0'
+ModuleVersion = '1.0.1'
```

Comment:
Please do not change the module version.  This is handled automatically by versioning tools

Reply:
And this as well